### DISCUSSION TITLE
Httpx - no screenshot, but a respons. Works in browser - SOLVED

### DISCUSSION THREAD

**Discussion by HachimanSec** [projectdiscovery/httpx#1374] [Q&A]
Hi everyone, 

I am running httpx after subfinder. Goal is to screenshot every identified site. 
This is the command I use:
httpx -ss -srd ./httpx-output

The versino is: httpx version v1.3.5

In the output I have the response folder, which contains .txt files for every host and the html response. 
The screenshot folder however is empty or in other runs it is missing a substantial amount of screenshots. 

If I run the same query multiple times, I receive different results. 

When I enter the website in the browser, I see the html rendered - even though it is just a status messagen that the file wasnt found. 

Any idea what this might be? 

PS: When I input a file, the result from subfinder, with 65 entries some pages are not screenshoted but HTTP response is logged. 
Do I explicitly run httpx for this missing - non screenshoted - hoste, it creates a screenshot. 

Can this be a performance / memory issue on my host?

RESULT: This appears to be a perf issue on my host. I scaled it up and now it seems to work.

Thx!

**Comment by MetzinAround** [comment]
Glad to see it worked out @HachimanSec ! 

Let us know if you have further questions!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
